components/markets/MarketsList.tsx

```diff
@@ -109,7 +109,7 @@ const MarketsList = observer(({ className = "" }: MarketsListProps) => {
   const [markets, setMarkets] = useState<
     (IndexedMarket<Context> & {
       outcomes: MarketOutcomes;
-      prediction: string;
+      prediction: string | [string, number];
```

Comment:
Are you sure all these union types are needed? I think it could just be changed to the new type that was added

Reply:
Changing the above (to return an object) sorted this problem out too now as I changed the types here